### DISCUSSION TITLE
teoria relazionale - 8

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#75] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/05bd85a2-870a-4ba0-8037-953b9506dfea)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/3c558ef9-1fb4-4009-b0f5-e7236720ac04)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/eff0ae1d-3b59-4a5b-8d5b-3e4ca5714fe6)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/89f83577-752d-409c-af4f-7a3e2267d068)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-24 175306](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/2382d0d3-9f0d-44c0-9c47-18edbe9db3e0)
![Immagine 2024-01-24 175320](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/03206593-968c-4cbb-b9b5-d1a74ba5a9a7)
![Immagine 2024-01-24 175332](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/ee34dc7d-9a1f-4428-b9c5-36c9c3c15176)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/8c878967-1a44-436f-b8a5-1aa96ef305b1)
